packages/main/test/specs/ComboBox.spec.js

```diff
@@ -283,6 +283,30 @@ describe("General interaction", () => {
 
 	});
 
+	it("should fire change event after the user has typed in value, but also selects it from the popover", async () => {
+        await browser.url(`test/pages/ComboBox.html`);
+
+		// Setup
+		const changeValue = await browser.$("#change-placeholder");
+        const counter = await browser.$("#change-count");
+        const combo = await browser.$("#change-cb");
+		const input = await combo.shadow$("[inner-input]");
+
```

Comment:
remove the line.

Reply:
Why? How would that improve the code or the behaviour?

Reply:
It improves the look of the code, since it's a rule of thumb to have one empty line between the different code blocks.
Also, I don't understand why once you use one empty line and after that you use two.

Reply:
You could divide the code into several abstract sections- setup of the scene, interaction, and assertions. I've tried that way to improve the readability as you've already written above. 
In the end, this is something very subjective and as we do not have a special rule (linter) for that, it's up to a personal preference. Furthermore, my belief is that such comments must not be part of a code review